compiler/main.bal

```diff
@@ -17,6 +17,7 @@ public type Options record {
     string? expectOutDir = ();
     string? gc = ();
     string? htmlError = ();
+    boolean outBir = false;
```

Comment:
We can't have `outBir` and `outDir`. Too lexically close but semantically different.